cli/sql_util.go

```diff
@@ -77,16 +76,18 @@ func printQueryOutput(rows *sql.Rows) error {
 
 	// Stringify all data and append rows to tablewriter.
 	vals := make([]interface{}, len(cols))
+	for i := range vals {
+		vals[i] = new(interface{})
```

Comment:
I'm not against using `new` where it makes sense, but have a definite preference for the `&foo{}` syntax for structure allocation. I was just being over-aggressive here.